src/analyzer/protocol/websocket/websocket.spicy

```diff
@@ -0,0 +1,143 @@
+# Copyright (c) 2024 by the Zeek Project. See LICENSE for details.
+
+module WebSocket;
+
+import spicy;
+import zeek;
+
+const OPCODE_CONTINUATION = 0x00;
+const OPCODE_TEXT         = 0x01;
+const OPCODE_BINARY       = 0x02;
+const OPCODE_CLOSE        = 0x08;
+const OPCODE_PING         = 0x09;
+const OPCODE_PONG         = 0x0a;
+
+# Spicy implementation of WebSocket unmasking.
+#
+# This isn't used because it's too slow. Instead, it's
+# implemented in C++ unmask.cc as fast_unmask().
```

Comment:
Do we need to keep this around?

```suggestion
# This isn't used because since this code is pretty hot.
# Instead we use `fast_unmask()` implemented in C++.
```